src/databricks/labs/ucx/install.py

```diff
@@ -282,18 +284,31 @@ def _confirm_force_install(self) -> bool:
         return False
 
     def configure(self, default_config: WorkspaceConfig | None = None) -> WorkspaceConfig:
+        connection_error_message = (
+            f"Cannot connect with {self.workspace_client.config.host} see "
+            "https://github.com/databrickslabs/ucx#network-connectivity-issues for help: %s"
+        )
         try:
             config = self.installation.load(WorkspaceConfig)
             self._compare_remote_local_versions()
-            if self._confirm_force_install():
-                return self._configure_new_installation(default_config)
-            self._apply_upgrades()
-            return config
+            if not self._confirm_force_install():
+                self._apply_upgrades()
+                return config
```

Comment:
```suggestion
            if not self._confirm_force_install():
                self._apply_upgrades()
            return config
```
otherwise a bug

Reply:
Thanks, it was not caught by my local test runs for some reason